test/DotsMobileStepper.test.tsx

```diff
@@ -0,0 +1,30 @@
+import { mount } from 'enzyme'
+import React from 'react'
+import { Button } from '@material-ui/core'
+import { DotsMobileStepper } from '../src/components/DotsMobileStepper'
+
+describe('handleNext', () => {
+  const proba = {
```

Comment:
```suggestion
  const mock = {
```
can you please use English variable names?

Reply:
Sure thing